lib/DxilContainer/DxilContainerAssembler.cpp

```diff
@@ -1531,6 +1530,12 @@ void hlsl::SerializeDxilContainerForModule(DxilModule *pModule,
   DXASSERT_NOMSG(pModule->GetSerializedRootSignature().empty());
 
   bool bMetadataStripped = false;
+
+  if (Flags & SerializeDxilFlags::StripReflectionFromDxilPart) {
+    pModule->StripReflection();
```

Comment:
If we strip reflection here, reflection will be removed for debug module too.
Is this expected?

Reply:
Should be fine. Debug info doesn't really depend on names and stuff.

Reply:
Never mind, we need the reflection data for debug info. We also don't want to strip names before writing RDAT (or at all, for libs anway).